From 647a3af126943b2251c82340ab7c81706b2eb805 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Fri, 27 Nov 2015 10:24:12 +0100
Subject: [PATCH] spi: fix build against MUSL C library

Without <linux/ioctl.h> include compiler cannot find _IOC_SIZEBITS
macro, when using MUSL C library.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 lib/spi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/spi.c b/lib/spi.c
index 8465f75..6d1f78a 100644
--- a/lib/spi.c
+++ b/lib/spi.c
@@ -6,6 +6,7 @@
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <linux/types.h>
+#include <linux/ioctl.h>
 #include <linux/spi/spidev.h>
 
 #include "libsoc_spi.h"
-- 
2.1.4

